docs/environment-variables.md

```diff
@@ -70,3 +71,48 @@ FLASK_ENVIRONMENT_CONFIG=dev
 SECRET_KEY='some_random_key'
 (...)
 ```
+
+# Database Environment Variables
+
+To run the backend in other than `local` mode i.e `prod` or `develop` you need to export database environment variables.
+These are the required database environment variables.
+```
+export DB_TYPE=<database_type>
+export DB_USERNAME=<database_username>
+export DB_PASSWORD=<database_password>
+export DB_ENDPOINT=<database_endpoint>
+export DB_NAME=<database_name>
+```
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 
+| DB_PASSWORD | Database password of the `DB_USERNAME` user| admin@123, mypwd123|
```

Comment:
```suggestion
| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |
```
maybe just adding a password example is less confusing, instead of using word1, word2, it can be confusing for contributors